tokens/mecha.json

```
@@ -0,0 +1,10 @@
+{
+  "ethereum_address": "",
+  "aurora_address": "0xa33C3B53694419824722C10D99ad7cB16Ea62754",
+  "name": "Mecha Tracker",
```

Token name should match metadata in ERC-20:

```suggestion
  "name": "Mecha",
```